.github/workflows/repo-stale.yaml

```diff
@@ -1,4 +1,4 @@
-name: Issue stale check
+name: Repo Stale
```

Comment:
Not a fan of the "Repo ..." names. Would be prefer something more descriptive like "Stale Issues Check" or "Check for stale issues".

Reply:
Capitalisation consistency on the titles is a bit weird too. If titles should be uppercase, all words should be.
We could theoretically follow the MusicBrainz English styleguide for titles ^^

Reply:
For consistency everything repository related changed to start with "Repo" so we now have App, Repo and Gradle

Reply:
For the files that's ok, but the title shown in GitHub is a downgrade imo. It's less easy to understand what's happening.